README.md

```diff
@@ -45,6 +45,10 @@ generate this data from a wikidata dump.
 
 ## Release notes
 
+### 2.3.0 (2015-06-26)
+* Replace use of Wikibase\TermIndex::getMatchingIDs with Wikibase\TermIndex::getTopMatchingTerms.
+* EntitySelector no longer passes "type" parameter to wbsgetsuggestions which avoids an "Unrecognized parameter" warning.
```

Comment:
So? This would still work with a 2.2.2 version number. This is not an actual problem, only a hypothetical question. But I still think a correct semver step would have been 0.0.1, not 0.1.0.